pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=61.0",
+    "setuptools_scm[toml]>=6.2"
+]
+
+[project]
+name = "opppy"
+version = "0.1.12"
```

Comment:
This looks great. Thanks @tylerjereddy! Do you think I should remove my "version.py" file as a follow on to this. I was mostly using this to track if I'm trying to append existing dictionary files that might have been parsed by pervious version of the code (probably overkill). Or should I just pull the version from this toml file in the python version.py that I currently have?

Reply:
Thanks for the example. I'm happy to delay changing the versioning (just updating the version in both places independently for now). I'll open a ticket based on this example and will address later if I find updating it in two places to error prone.